cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -3335,6 +3332,32 @@ fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(
             }
         }
 
+        Opcode::Bint => {
+            // Booleans are stored as all-zeroes (0) or all-ones (-1). We AND
+            // out the LSB to give a 0 / 1-valued integer result.
+            let rn = put_input_in_reg(ctx, inputs[0]);
+            let rd = get_output_reg(ctx, outputs[0]);
+            let ty = ctx.output_ty(insn, 0);
+
+            ctx.emit(Inst::imm(OperandSize::Size64, 1, rd.regs()[0]));
+            ctx.emit(Inst::alu_rmi_r(
+                OperandSize::Size64,
+                AluRmiROpcode::And,
+                RegMemImm::reg(rn),
```

Comment:
Can you use an immediate operand here for the constant 1, instead?

Reply:
Ah, yeah, dunno why I did it this way instead -- fixed!